From 9e2490713742a882903e4af782568c5d5d009868 Mon Sep 17 00:00:00 2001
From: Dave Rothlisberger <dave@rothlis.net>
Date: Tue, 16 Sep 2008 09:53:33 -0500
Subject: [PATCH] Removed duplicate unit test.
 The functionality in the removed test is already covered by
 test_nested_fields_for_with_index_on_both.

---
 actionpack/test/template/form_helper_test.rb |   16 ----------------
 1 files changed, 0 insertions(+), 16 deletions(-)

diff --git a/actionpack/test/template/form_helper_test.rb b/actionpack/test/template/form_helper_test.rb
index 52e8bf3..473ba76 100644
--- a/actionpack/test/template/form_helper_test.rb
+++ b/actionpack/test/template/form_helper_test.rb
@@ -464,22 +464,6 @@ class FormHelperTest < ActionView::TestCase
   end
 
   def test_nested_fields_for_with_index
-    form_for('post', @post, :index => 1) do |c|
-      concat c.text_field(:title)
-      c.fields_for('comment', @comment, :index => 1) do |r|
-        concat r.text_field(:name)
-      end
-    end
-
-    expected = "<form action='http://www.example.com' method='post'>" +
-               "<input name='post[1][title]' size='30' type='text' id='post_1_title' value='Hello World' />" +
-               "<input name='post[1][comment][1][name]' size='30' type='text' id='post_1_comment_1_name' value='new comment' />" +
-               "</form>"
-
-    assert_dom_equal expected, output_buffer
-  end
-
-  def test_nested_fields_for_with_index
     form_for(:post, @post, :index => 1) do |f|
       f.fields_for(:comment, @post) do |c|
         concat c.text_field(:title)
-- 
1.5.6.4

